webapp/api-doc/LambdaInstanceStatus.md

```diff
@@ -47,7 +47,7 @@ In this example we are going to get the status of the lambda instance with uuid
 The request in curl
 
 ```
-    curl -kgX GET -H 'x-api-key: tJ3b3f32f23ceuqdoS_TH7m0d6yxmlWL1r2ralKcttY' -G 'http://<url>:<port>/backend/lambda-instances/3/status'
+    curl -X GET -H "x-api-key: tJ3b3f32f23ceuqdoS_TH7m0d6yxmlWL1r2ralKcttY" 'http://<url>:<port>/backend/lambda-instances/3/status'
```

Comment:
a couple of suggestions:

- remove the 4 spaces here and elsewhere in the .md files
- remove also the `:<port>` part from the url (here and elsewhere in the docs).